Source/Scene/ArcGisMapServerImageryProvider.js

```diff
@@ -56,6 +56,7 @@ define([
      *
      * @param {Object} options Object with the following properties:
      * @param {String} options.url The URL of the ArcGIS MapServer service.
+     * @param {String} [options.securityToken] The ArcGIS token used to authenticate with the ArcGIS MapServer service.
```

Comment:
Looking at the ESRI documentation, they never seem to use the term `security token`, so I would just rename this to just `token` everywhere, since that's always how they refer to it.